src/test/groovy/graphql/nadel/NadelTest.groovy

```diff
@@ -63,8 +61,6 @@ class NadelTest extends Specification {
         0 * graphqlRemoteRetriever1.queryGraphQL(*_) >> completedFuture([data: []])
     }
 
-    //fime: get this up and running again once DSL refactoring is over
-    @Ignore
     def "stitching with transformation"() {
```

Comment:
we should also use hydration here

Reply:
what do you mean? hydration is tested here (tbh it is the only thing that is tested :) )

Reply:
I mean the name of the test :-)